### ISSUE TITLE
Error/warning while building in VS

### ISSUE THREAD

**Issue by JasonShearer** [snesrev/zelda3#150]
### Describe your bug here. And how to reproduce it.

It built fine as of 2cfb4cb, but something broke since then and now I'm getting this error/warning as of 8f4b242.

![image](https://user-images.githubusercontent.com/14334181/195956476-26a797ca-3e3a-42ae-a42e-5d896d62d7f4.png)

### What is your build target?

Windows

**Comment by krishean** [comment]
I'm also seeing these errors with VS 2022 Build Tools:

```
Build FAILED.

"Zelda3.sln" (Clean;Build target) (1) ->
"zelda3.vcxproj" (default target) (2:2) ->
(ClCompile target) ->
  C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [zelda3.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [zelda3.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [zelda3.vcxproj]


"Zelda3.sln" (Clean;Build target) (1) ->
"zelda3.vcxproj" (default target) (2:2) ->
(ClCompile target) ->
  C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531,5): error C2220: the following warning is treated as an error [zelda3.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531,5): error C2220: the following warning is treated as an error [zelda3.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\winbase.h(9531,5): error C2220: the following warning is treated as an error [zelda3.vcxproj]

    3 Warning(s)
    3 Error(s)

Time Elapsed 00:00:17.54
```

**Comment by snesrev** [comment]
Can you check if this is fixed in bf30ca05480103c118ef4503f2884cff9334e6dc

**Comment by krishean** [comment]
Successfully builds with VS 2022 Build Tools as of bf30ca0
